frontend/src/scenes/insights/views/InsightsTable/InsightsTable.tsx

```diff
@@ -271,6 +271,9 @@ export function InsightsTable({
                 let value: number | undefined = undefined
                 if (calcColumnState === 'total' || isDisplayModeNonTimeSeries) {
                     value = item.count ?? item.aggregated_value
+                    if (item.aggregated_value > item.count) {
+                        value = item.aggregated_value
+                    }
```

Comment:
Ah, perhaps we should be using here `NON_TIME_SERIES_DISPLAY_TYPES`. Display types in that list use `aggregated_value`, while the others depend on `count` AFAIK. Though it's hard to be certain without test coverage – clearly there's none here…

Reply:
Let's get this fix in for our users first and look at a longer term solution afterwards 😃